starlette/starter.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+from temporalio.client import Client
+from worker import SayHello
+
+from starlette.applications import Starlette
+from starlette.responses import JSONResponse
+from starlette.routing import Route
+
+
+async def say_hello_endpoint(request):
+    client = await Client.connect("localhost:7233")
```

Comment:
Similar to the other PR at #39, we need to not connect a client on every endpoint but rather reuse one. This needs to use the accepted way to add/inject stateful dependencies into request handlers. https://www.starlette.io/applications/#storing-state-on-the-app-instance may help.